events/certificates.py

```diff
@@ -0,0 +1,128 @@
+from django.conf import settings
+from spoken.config import EDUPYRAMIDS_CERTIFICATE_DATE
+import re
+import os
+
+SCHOOL = 24
+FDP = 169
+CSC = 18
+
+def get_training_certificate(ta):
+   """
+      return training certificate template path
+      ta : TrainingAttend obj
+   """
+   spk = {
+      'stp': 'Blank-Certificate.pdf',
+      'fdp': 'fdptr-certificate.pdf',
+      'csc': 'Certificate_CSC_blank.pdf'
+   }
+   edupyramids = {
+      'stp': 'Blank-Certificate_edupyramids.pdf',
+      'fdp': 'fdptr-certificate_edupyramids.pdf',
+      'csc': 'Certificate_CSC_blank_edupyramids.pdf'
+   }
+   if ta.training.department.id == FDP:
+      cert_type = 'fdp'
+   elif ta.training.training_planner.academic.institution_type_id == CSC:
+      cert_type = 'csc'
+   else:
+      cert_type = 'stp'
+   
+   if ta.training.training_start_date < EDUPYRAMIDS_CERTIFICATE_DATE:
+      return os.path.join(settings.MEDIA_ROOT, spk[cert_type])
+   return os.path.join(settings.MEDIA_ROOT, edupyramids[cert_type])
+
+def get_test_certificate(ta):
+   """
+      return test certificate template path
+      ta : TestAttendance obj
+   """
+   spk = {
+      'stp': 'Blank-Certificate.pdf',
+      'fdp': 'fdp-test-certificate.pdf',
+      'csc': 'Certificate_CSC_blank.pdf'
+   }
+   edupyramids = {
+      'stp': 'Blank-Certificate_edupyramids.pdf',
+      'fdp': 'fdp-test-certificate_edupyramids.pdf',
+      'csc': 'Certificate_CSC_blank_edupyramids.pdf'
+   }
```

Comment:
merge with the global structure above. I think only the fdp key has a different value. Use a different key when merging.